src/dictionaries/proper_nouns

```diff
@@ -0,0 +1,40 @@
+Aldergrove
+Boolean
+Booleans
+Camborne
+conda
+Conda
```

Comment:
?

Reply:
What are you questioning?

> Why both upper and lower case (programmatic)?

Conda gets used as a command as well as a proper noun. Without more sophisticated context we need to allow for both.

> Why both upper and lower case (grammatical)?

You need both, because I'm not attempting to auto generate the sentence case versions of words in the "proper_nouns" list. You have to explicitly state proper nouns you want to allow a lower case version of.

Reply:
> Conda gets used as a command as well as a proper noun. Without more sophisticated context we need to allow for both

The same applies to Cylc but we only have the capitalised entry:

```rst
To open Cylc TUI run the command ``cylc tui``.
```

Presumably Conda should be held to the same standard?

Reply:
> The same applies to Cylc but we only have the capitalised entry:

Not true, I just put "cylc" in the word list by mistake.